nix/config.nix

```diff
@@ -1 +1,4 @@
-{}
+{ config, ...}: {
+  # tasty-hedgehog-coverage wasn't building due to tasty deps not being met.
+  packages.tasty-hedgehog-coverage.components.library.doExactConfig = true;
```

Comment:
What is this line doing?

Reply:
And is this something we might want to document in case somebody runs into a similar problem in the future?